owasm/src/ext/utils/date.rs

```diff
@@ -0,0 +1,41 @@
+use crate::core::{Oracle, ShellCmd};
+
+pub struct Date {}
+
+impl Date {
+    pub fn new() -> Date {
+        Date {}
+    }
+}
+
+impl Oracle for Date {
+    type T = u64;
+
+    fn as_cmd(&self) -> ShellCmd {
+        ShellCmd::new("date", &["+'%s'"])
+    }
+
+    fn from_cmd_output(&self, output: String) -> Option<u64> {
+        output.replace("\n", "").replace("'", "").parse::<u64>().ok()
```

Comment:
Why is `replace("'", "")` necessary? What if we do `ShellCmd::new("date", &["+%s"])`?